modules/swagger-codegen/src/main/java/config/ConfigParser.java

```diff
@@ -7,11 +7,16 @@
 import java.util.Iterator;
 import java.util.Map;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public class ConfigParser {
 
+    private static final Logger LOG = LoggerFactory.getLogger(ConfigParser.class);
+
     public static Config read(String location) {
 
-        System.out.println("reading config from " + location);
+    	LOG.debug("reading config from " + location);
```

Comment:
@rostskadat thanks for the PR. For consistency, can we use LOGGER instead of LOG?

Also there seems to be some formatting issue in this files and other files, e.g. above line is missing one space.